testplan/web_ui/testing/src/AssertionPane/AssertionTypes/GraphAssertions/App.css

```diff
@@ -0,0 +1,33 @@
+.App {
```

Comment:
Could you use Aphrodite? Let's keep CSS styles in same file as component. Makes each component a bit more readable.